production/jaeger-production-template.yml

```diff
@@ -196,8 +89,10 @@ objects:
             protocol: TCP
           command:
           - "/go/bin/collector-linux"
-          - "--cassandra.servers=cassandra"
-          - "--cassandra.keyspace=${KEYSPACE}"
+          - "--span-storage.type=${JAEGER_STORAGE_TYPE}"
+          - "--cassandra.servers=cassandra" # ignored if Cassandra is not used
+          - "--cassandra.keyspace=${KEYSPACE}" # ignored if Cassandra is not used
+          - "--es.server-urls=http://elasticsearch:9200" # ignored if ES is not used
```

Comment:
Would it be better to have the URL defined in an env var aswell, in case the user already has their own ES cluster somewhere? With this value as the default.